typing/typedtree.ml

```diff
@@ -342,15 +342,20 @@ and module_type_declaration =
      mtd_loc: Location.t;
     }
 
-and open_description =
+and 'a open_infos =
     {
-     open_path: Path.t;
-     open_txt: Longident.t loc;
+     open_expr: 'a;
+     open_type: Types.signature;
```

Comment:
This could use a clearer name like `open_bound_items`. `open_type` sounds like it always contains the type of the opened module, but for the optimised case it is actually just empty.